solve.py

```diff
@@ -0,0 +1,51 @@
+import os.path
+
+def grab_input(rosalind_problem_name, downloads_directory):
+    """
+    Opens a text file representing rosalind input to solve a problem.
+    Pass in the name of the current rosalind problem name to look it up
+    in your downloads directory, where rosalind text files typically end up.
+
+    >>> grab_input('rosalind_ini3.txt')
+    wpoR2sT77ULTxwXaC9DNJKsM2jSUFlaDvhySPNATyhO92brubeDVnRXnx6YYzS88UHolodact
+    ylusbDN4pAeMJiGTF0IWa0u8VSWrBcJXmog5JI0kDVyklRxUxkCZihLQ1aJwqFhemilasiusw
+    hlfQ1AxRYA6H3QNFTcNnNiWfn7cuE.
+    65 76 135 144
+    """
+    if downloads_directory is None:
+        downloads_directory = '~/Downloads'
+    downloads_directory = os.path.expanduser(downloads_directory)
```

Comment:
Yes, the "tilde" character `~` is used on the command line to represent a user's home directory. In unix variants (this includes Mac), this is `/home/rbcaperson/` or whatever.

In windows, I'm pretty sure it's `C:\Users\rbcaperson\`....I'd have to look it up. Basically, it's a cross-platform way to get to places like the desktop, the downloads directory, etc.